lib/graphql/query/base_execution/value_resolution.rb

```diff
@@ -20,6 +20,11 @@ def initialize(value, field_type, target, parent_type, ast_field, query, executi
           end
 
           def result
+            return nil if value.nil?
```

Comment:
I avoid ternaries when the condition ends in `?` because I find the double-`?` hard to read